resources/src/main.rs

```diff
@@ -65,10 +65,18 @@ async fn intercept_command(body: &Value) {
     let channel: &str = body["event"]["channel"]
         .as_str()
         .unwrap_or("invalid_channel");
+    if channel != "C0351GJ62Q0" {
+        log::info!("This channel is not the test channel");
+        return;
+    }
```

Comment:
For prod we are going to be writing to more than just the test channel. This check is fine, but if we want to keep it we should add a prod vs dev config. You could add another env var of "list of allowed channels" or something in `lib/devil-bot-rust-cdk-stack.ts`.

Reply:
Yes, thinking of adding some list of allowed channels